blockly/apps/blocklyduino/blockly_helper.js

```diff
@@ -38,14 +38,16 @@ function restore_blocks() {
 function save() {
   var xml = Blockly.Xml.workspaceToDom(Blockly.mainWorkspace);
   var data = Blockly.Xml.domToText(xml);
-
+  var fileName = window.prompt("What would you like to name your file?", "BlocklyDuino");
```

Comment:
This file uses single quotation marks (`'`), so I'd suggest to use the same convection.